src/libos/src/lib.rs

```diff
@@ -53,6 +53,8 @@ extern crate serde;
 extern crate serde_json;
 #[macro_use]
 extern crate memoffset;
+#[macro_use]
```

Comment:
I didn't notice that we need any macros from the resolve_conf crate, do we?

Reply:
Done, we don't use any macros from resolv_conf crate.